crates/core/c8y_translator/fuzz/fuzz_targets/fuzz_target_1.rs

```diff
@@ -1,8 +1,8 @@
 #![no_main]
 use libfuzzer_sys::fuzz_target;
 
-use c8y_translator::CumulocityJson;
+use c8y_translator::json;
 
 fuzz_target!(|data: &[u8]| {
-    let _ = CumulocityJson::from_thin_edge_json(data);
+    let _ = json::from_thin_edge_json(std::str::from_utf8(data).unwrap());
```

Comment:
@didier-wenzek Seeing that this project wasn't even compiling for a while (because it's not even built as part of our workflows), was that because this project isn't relevant anymore or if it is, should we make sure that it builds as part of our workflows?